### PR TITLE
Update 02_how-do-i-delete-my-account-profile.md

### PR DESCRIPTION
**Pull request by sylee-**

### CODE REVIEW

**Review by Haribishop — APPROVED** [review]

**Review by emfet — APPROVED** [review]
Looks good, but I think for this one we may also want to stress that deletion cannot be reversed. I get a lot of emails from ppl asking if we can reactivate a deleted profile.

File: 02_profiles/02_how-do-i-delete-my-account-profile.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 # How do I delete my account/profile?
 
-The easiest way to delete your profile is to go here: [https://fetlife.com/deactivate?type=permanently%20delete](https://fetlife.com/deactivate?type=permanently%20delete)
+You can delete your profile [here](https://fetlife.com/deactivate?type=permanently%20delete). 
 
 Alternatively, follow these steps to delete or deactivate your profile:
+1. Select the hamburger menu (icon with three horizontal lines) on the far right of FetLife"s navigation bar.
+2. Select “Update Settings" in the dropdown menu.
+3. On the right side of the page, you'll see a "Deactivate/Delete Account” section.
+4. Click on “Deactivate or Delete my Account”.
```
Comment (DarklingMuse) [inline 02_profiles/02_how-do-i-delete-my-account-profile.md:9]: Replace 'click' with select as per Nielsen's previous recommendation for language that isn't device specific

File: 02_profiles/02_how-do-i-delete-my-account-profile.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+5. Select “Delete” from the dropdown box that appears.
+6. Finally, enter your password and submit the form. Your profile will then be put into the delete queue.
 
-1. Click on the hamburger menu (icon with three horizontal lines) on the far right of FetLife's navigation bar.
-2. Click on “Update Settings" in the pop out menu.
-3. On the right side, you'll see a link to "Deactivate/Delete Account".
-4. Select Delete from the dropdown box, and confirm your password.
+When you set your profile for deletion, it becomes inactive and is put into the queue to be deleted in seven days. Change your mind? No worries! Just log back into FetLife anytime in the next seven days. Your profile will become active and move out of the delete queue!
```
Comment (DarklingMuse) [inline 02_profiles/02_how-do-i-delete-my-account-profile.md:13]: I would suggest phrasing it as 'When you set your profile for deletion, it is removed from the website and your data is put into our server queue to be deleted in seven days.' - I've seen members get upset thinking that their profile is hanging around for 7 days so I think being clear that it is removed from the website / public view is a good thing!

Also 'changed your mind' not change.

To make the last line a bit more concise I'd go with just: 'Just log back into FetLife anytime in the next seven days and your profile will become active again.'

**Review by DarklingMuse — COMMENTED** [review]
suggestions added as comments!

**Comment by DarklingMuse** [comment]
> Looks good, but I think for this one we may also want to stress that deletion cannot be reversed. I get a lot of emails from ppl asking if we can reactivate a deleted profile.

I agree!

File: 02_profiles/02_how-do-i-delete-my-account-profile.md (hunk 1 above)
Comment (sylee-) [reply 02_profiles/02_how-do-i-delete-my-account-profile.md]: Done.

File: 02_profiles/02_how-do-i-delete-my-account-profile.md (hunk 2 above)
Comment (sylee-) [reply 02_profiles/02_how-do-i-delete-my-account-profile.md]: I've made the changes suggested for the last two paragraphs.

I tried to follow the suggestion of the first paragraph.  I'm not sure it's in the right place or if it's even good enough.

**Review by DarklingMuse — APPROVED** [review]
👍

**Pull request closed without merging**